sky/utils/schemas.py

```diff
@@ -155,6 +155,69 @@ def get_storage_schema():
     }
 
 
+def get_service_schema():
```

Comment:
docstr

Reply:
What kind of docstr..? There is no docstr for other functions in the file so I'm not sure what information you think we need to include here 😵‍💫

Reply:
"""Schema for top-level `service:` field (for SkyServe)."""